### ISSUE TITLE
Deprecation notices > `getExcludedExceptions` is deprecated since version 2.3 

### ISSUE THREAD

**Issue by ruudk** [getsentry/sentry-symfony#294]
Just installed Sentry 2.3.0 and getting this deprecation notices that error on my CI:

```
Method Sentry\Options::getExcludedExceptions() is deprecated since version 2.3 and will be removed in 3.0. Use the "IgnoreErrorsIntegration" integration instead.
```

**Comment by ruudk** [comment]
More deprecations:
```
The option "project_root" is deprecated. Please use the "in_app_include" option instead.
```

**Comment by Jean85** [comment]
Sorry, I'm a bit behind with the release schedule. I'll release a 3.3.0 ASAP with all that's in master, and the attack those issues, that are popping from the new release of `sentry/sentry`, the 2.3.0

**Comment by tristanbes** [comment]
thanks @Jean85 we're also interested in this release

**Comment by julienanne** [comment]
Hi, also interested in this release

**Comment by Jean85** [comment]
#298 should fix all of those. Can you test it out?

**Comment by geerteltink** [comment]
I can confirm that mentioned deprecation notices are gone with that PR.

**Comment by julienanne** [comment]
thanks

**Comment by Jean85** [comment]
Issue is resolved in #298 and released in 3.4.0.
